modules/logging-identity-federation.adoc

```diff
@@ -0,0 +1,58 @@
+
+// Module included in the following assemblies:
+//
+// * list of assemblies where this module is included
+// logging/log_storage/installing-log-storage.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="<logging-identity-federation>_{context}"]
+= Identity Federation
```

Comment:
Should this be the following or is Identity  Federation a proper noun (which might need fixing throughout the docs)??
```suggestion
= Identity federation
```